### ISSUE TITLE
job-ingest: allow module options to be set via configuration

### ISSUE THREAD

**Issue by grondo** [flux-framework/flux-core#3541]
In #3533 a new job validator was introduced to allow multiple validators to run simultaneously. After that PR is merged, however, the list of validator "plugins" used by the ingest module and any other options can only be provided on the module load command line, e.g. `flux module load job-ingest validator-plugins=jobspec,feasibility`.

Support for a TOML config for job-ingest should be added, especially for use with the system instance, e.g.

```toml
[job-ingest.validator]
plugins = [ "jobspec", "feasibility" ]
args = [ "--require-version=1" ]
```

**Comment by chu11** [comment]
solved by #4238 ?

**Comment by grondo** [comment]
Yeah, thanks! I had apparently opened a duplicate issue #4227
